Sources/MapboxMaps/Foundation/CoordinateBoundsAnimator.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import MapboxCoreMaps.CoordinateBounds
+import Turf
+
+internal protocol CoordinateBoundsAnimator {
+    func show(coordinateBounds: CoordinateBounds, padding: UIEdgeInsets, pitch: CGFloat?, animationDuration: TimeInterval)
```

Comment:
Nit: We do not have a practice of documentation for internal/private methods but can you add a few words for this one?